task/hiffy/Cargo.toml

```diff
@@ -41,6 +41,7 @@ testsuite = [ "test-api" ]
 i2c = []
 gpio = []
 spi = []
+send = []
```

Comment:
I think we want this enabled on every non-G030 board; otherwise, `humility hiffy -c Foo.bar ...` won't work.  Enabling it with the `stm32h7` and `lpc55` (and `stm32g031`?) features is probably the easiest way to do this.

Alternatively, we could use the `micro` feature to _also_ mean that the `send*` functions should be disabled.  This breaks the "purely additive feature" rule, though...

Reply:
Note that the send code is not _optional_ on stm32h7/lpc55/etc, only on g0, rn. But I can add the send feature to the others like it does something if that'd be clearer.